assets/stylesheet.css

```diff
@@ -0,0 +1,38 @@
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: Helvetica, Arial, sans-serif;
+    color: powderblue;
+    font-weight: bolder;
+    font-variant: small-caps;
+    margin-left: 25px; 
+    margin-top: 50px;
+    margin-right: 25px;
+
+}
+p {
+    font-family: Verdana, Arial, serif;
+    font-size: 13px;
```

Comment:
It is better practice to use relative units (eg. rem or em) instead of fixed units (i.e. px) for font sizes.

<img width="788" alt="image" src="https://github.com/user-attachments/assets/cc322854-f524-496b-933b-321fdcd996fc" />

If you are curious, you can probably read more about it in the [MDN Docs](https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Values_and_units#numbers_lengths_and_percentages)

This is the official docs for HTML, CSS, and Javascript, so I highly recommend treating this like the bible haha.

Reply:
This is not a blocker to merging this PR btw.